content/pt-br/docs/reference/setup-tools/kubeadm/generated/kubeadm_join.md

```diff
@@ -0,0 +1,48 @@
+<!--
+The file is auto-generated from the Go source code of the component using a generic
+[generator](https://github.com/kubernetes-sigs/reference-docs/). To learn how
+to generate the reference documentation, please read
+[Contributing to the reference documentation](/docs/contribute/generate-ref-docs/).
+To update the reference content, please follow the 
+[Contributing upstream](/docs/contribute/generate-ref-docs/contribute-upstream/)
+guide. You can file document formatting bugs against the
+[reference-docs](https://github.com/kubernetes-sigs/reference-docs/) project.
+-->
+
+
+Rode este comando em qualquer máquina que você deseje adicionar a um cluster
+existente
+
+### Sinopse
+
+
+
+Ao associar um novo nó a um cluster inicializado com kubeadm, temos que
+estabelecer confiança bidirecional. Este processo é dividido entre a descoberta
+(em que o nó estabelece confiança na camada de gerenciamento do Kubernetes) e a
+inicialização TLS (em que a camada de gerenciamento do Kubernetes estabelece
+confiança no nó).
+
+Existem duas principais formas de descoberta. A primeira delas é o uso de um
+token compartilhado, juntamente com o endereço IP do servidor da API. A segunda
+é o fornecimento de um arquivo - um subconjunto do arquivo kubeconfig padrão. O
+arquivo de descoberta/kubeconfig suporta autenticação por token, plugins de
+autenticação do client-go ("exec"), "tokenFile" e "authProvider". Este arquivo
+pode ser um arquivo local ou um arquivo baixado através de uma URL HTTPS. Os
+formatos são `kubeadm join --discovery-token abcdef.1234567890abcdef 1.2.3.4:6443`,
+`kubeadm join --discovery-file caminho/para/arquivo.conf`, ou
+`kubeadm join --discovery-file https://endereco/arquivo.conf`. Somente um formato
+pode ser utilizado. Se os dados para a descoberta são carregados de uma URL,
+HTTPS deve ser utilizado. Neste caso, o conjunto de CAs instalado no host é
```

Comment:
sounds better to me "o HTTPS deve ser utilizado".

Reply:
Interesting. I tend to think that this kind of thing is uncountable, hence not prefixed by an article.

Do you think it should be "o protocolo HTTPS" or really just "o HTTPS"?

Reply:
UAU! It is complete and better "o protocolo HTTPS".

Reply:
Updated!